index.html

```diff
@@ -4975,6 +4975,621 @@ <h2 id="AudioWorkerParamDescriptor">
           </section>
         </section>
       </section>
+      <section>
+        <h2 id="AudioWorklet">
+          The <dfn>AudioWorklet</dfn> interface
+        </h2>
+        <p>
+          The AudioWorklet object allows importing user-supplied Javascript code
```

Comment:
I think the style is to use either `<code>AudioWorklet</code>` or `<a><code>AudioWorklet</code></a>` if you want a link.